sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -27,11 +27,12 @@ import {
   ReceivedMessageInfo,
   ServiceBusMessage,
   SendableMessageInfo,
-  DispositionStatus
+  DispositionStatus,
+  toAmqpMessage
```

Comment:
Question: Should we rename this to `ToAmqpMessage`?

Reply:
All functions follow camel casing, class/interface/enum follow pascal casing. So by that, no, we shouldnt change this to `ToAmqpMessage`